src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -522,7 +522,7 @@ static List<T> Goo<T>(T t)
             var expectedOrderedItems = new List<SignatureHelpTestItem>
             {
                 new SignatureHelpTestItem(
-$@"void List<'a>.Add('a item)
+$@"void List<'a?>.Add('a? item)
```

Comment:
`goo` should be non-null `'a`.

Reply:
Good catch, the nullability of `var` declarations should be determined by the initializer. I've added a test, `StaticNullChecking.LocalVar_NonNull`, and I'll address this in a subsequent PR.